File: legacy-runtime/src/main/java/com/xiaomi/common/logger/thrift/mfs/LandNodeInfo.java
package com.xiaomi.common.logger.thrift.mfs;

import java.util.Map;

public class LandNodeInfo {
    private String ip;
    private Map<String, Integer> expInfo;
    private int successCount;
    private int failedCount;
    private long duration;
    private int size;

    public void setIp(String ip) {
        this.ip = ip;
    }

    public void setExp_info(Map<String, Integer> expInfo) {
        this.expInfo = expInfo;
    }

    public void setSuccess_count(int successCount) {
        this.successCount = successCount;
    }

    public void setFailed_count(int failedCount) {
        this.failedCount = failedCount;
    }

    public void setDuration(long duration) {
        this.duration = duration;
    }

    public void setSize(int size) {
        this.size = size;
    }
}

File: legacy-runtime/src/main/java/com/xiaomi/common/logger/thrift/mfs/LandNodeInfo.kt
package com.xiaomi.common.logger.thrift.mfs

class LandNodeInfo {
    private var ip: String? = null
    private var expInfo: Map<String, Int>? = null
    private var successCount = 0
    private var failedCount = 0
    private var duration = 0L
    private var size = 0

    fun setIp(ip: String?) {
        this.ip = ip
    }

    fun setExp_info(expInfo: Map<String, Int>?) {
        this.expInfo = expInfo
    }

    fun setSuccess_count(successCount: Int) {
        this.successCount = successCount
    }

    fun setFailed_count(failedCount: Int) {
        this.failedCount = failedCount
    }

    fun setDuration(duration: Long) {
        this.duration = duration
    }

    fun setSize(size: Int) {
        this.size = size
    }
}
